torch/_inductor/cudagraph_trees.py

```diff
@@ -1519,13 +1520,11 @@ def _allocate_and_copy_recording_inputs(
                 elif i not in self.static_input_idxs:
                     # static_input does an allocation!
                     recording_inputs.append(static_input(inp))
-                    # copy over and clear non recording input
-                    self._copy_input(i, recording_inputs[-1], inp)
-                    inputs[i] = None
-                    del inp
```

Comment:
I figured this `del` wasn't that important, since the function finishes shortly after this for loop ends, and the `inp` variable will go out of scope at that point.